app/javascript/Typings.ts

```diff
@@ -31,6 +31,7 @@ export interface Dependency {
   name: string
   project_id: number
   version: string
+  update_required?: boolean
```

Comment:
This casing is inconsistent with the JS/JSON-style casing elsewhere in this file. Intentional?

Reply:
We are not parsing this data after returning to the db, just sending as json so expected. All the values we are creating/manually assigning to send to the client do use the camel convention.